doc/whats_new.rst

```diff
@@ -58,6 +60,8 @@ API
 
     - :func:`mne.io.read_raw_brainvision` now has ``event_id`` argument to assign non-standard trigger events to a trigger value by `Teon Brooks`_
 
+    - :func:`mne.read_epochs` now has ``add_eeg_ref=False`` by default, since average EEG reference can be added before writing or after reading using the method :func:`mne.Epochs.add_eeg_ref`.
+
```

Comment:
Had a brief chat with @agramfort a bit ago and he agreed, so I'll make a PR for that once 0.11 is out